### PR TITLE
Add link to try.gitea.io

### PR DESCRIPTION
**Pull request by strk**
.. also mention planned self-hosting and add link to download area

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

File: content/page/index.en-us.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ draft: false
 
 # Gitea - Git with a cup of tea
 
-Gitea is a community managed fork of [Gogs](http://gogs.io), lightweight code hosting solution written in [Go](https://golang.org/) and published under the [MIT](https://github.com/go-gitea/gitea/blob/master/LICENSE) license. The code of Gitea is currently hosted on [GitHub](https://github.com/go-gitea/).
+Gitea is a community managed fork of [Gogs](http://gogs.io), lightweight code hosting solution written in [Go](https://golang.org/) and published under the [MIT](https://github.com/go-gitea/gitea/blob/master/LICENSE) license.
 
-At the time of writing, there are no official Gitea release yet, but we're working hard to get to 1.0.0 in a way that would allow seamless upgrades of existing Gogs deploys.
+You can [try it online](https://try.gitea.io/) to get a feel of what it can do for you.
+
+The code of Gitea is currently hosted on [GitHub](https://github.com/go-gitea/)but we plan to self-host it as soon as possible.
```
Comment (tboerger) [inline content/page/index.en-us.md]: Missing space

File: content/page/index.en-us.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ draft: false
...
+
+At the time of writing, there are no official Gitea release yet, but we're working hard to get to 1.0.0 in a way that would allow seamless upgrades of existing Gogs deploys; keep an eye on the [download](https://dl.gitea.io) section
```
Comment (tboerger) [inline content/page/index.en-us.md]: > there are no official Gitea release yet

?

Comment (tboerger) [reply content/page/index.en-us.md]: I would drop `; keep an eye on the [download](https://dl.gitea.io) section`

**Comment by tboerger** [comment]
Generally I prefer lesser new lines as all the single sentences will be transferred to separate paragraphs.

File: content/page/index.en-us.md (hunk 2 above)
Comment (strk) [reply content/page/index.en-us.md]: This hasn't changed in this PR

**Comment by strk** [comment]
Given the amount of current content, it isn't bad to have more paragraphs..

File: content/page/index.en-us.md (hunk 1 above)
Comment (strk) [reply content/page/index.en-us.md]: space added

**Comment by tboerger** [comment]
Instead of stretching simple phrases we want things like screenshots.

**Comment by strk** [comment]
I think we can do better incrementally, how about merging this and
sending a new PR for next iteration ?

**Comment by bkcsoft** [comment]
I would flip the 2 lower around and then put them in the same paragraph.

**Comment by tboerger** [comment]
:-1: stop the line wrapping.

**Comment by strk** [comment]
How do line wrapping hurt ?
What does your -1 mean in terms of acceptability of this PR ?
The CONTRIBUTING.md file doesn't mention veto powers nor does
go into details of acceptance/rejection (it probably should)

**Comment by lunny** [comment]
All are right except line wrapping. It will break the UI on mobile or other small screen devices.

**Comment by strk** [comment]
On Mon, Dec 19, 2016 at 05:12:51PM -0800, Lunny Xiao wrote:
> All are right except line wrapping. It will break the UI on mobile or other small screen devices.

Single linebreaks in markdown have no structural meaning and should
thus not change the rendered output in any way. Do you have proof of
your statement about breaking on mobile UI and small screen devices ?

**Comment by tboerger** [comment]
Just revert the line breaks to match the style of ALL other markdown documents. Without this revert I'm against this pr.

**Comment by strk** [comment]
Not needed anymore now, there's a big green TRY button now..

**Pull request closed without merging**